{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "fc3af258_f7e094b7",
        "filename": "functions",
        "patchSetId": 3
      },
      "lineNbr": 149,
      "author": {
        "id": 13252
      },
      "writtenOn": "2023-12-11T08:50:05Z",
      "side": 1,
      "message": "Maybe also use \"Download failed:\" here for consistency with the previous attempts?\n\nAlso I\u0027m wondering whether we should error out in this case instead of silently returning and waiting for devstack to fail in some later stage?",
      "range": {
        "startLine": 149,
        "startChar": 30,
        "endLine": 149,
        "endChar": 39
      },
      "revId": "6b0f055b4ed407f8a190f768d0e654235ac015dd",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "de8f9867_ffec5681",
        "filename": "functions",
        "patchSetId": 3
      },
      "lineNbr": 149,
      "author": {
        "id": 13861
      },
      "writtenOn": "2023-12-11T08:57:28Z",
      "side": 1,
      "message": "Actually not changed that intentionally as this patch focused only on retry attempts in case of failures, but if there is strong ask can be changed in follow up?",
      "parentUuid": "fc3af258_f7e094b7",
      "range": {
        "startLine": 149,
        "startChar": 30,
        "endLine": 149,
        "endChar": 39
      },
      "revId": "6b0f055b4ed407f8a190f768d0e654235ac015dd",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}